backend/test/db.test.ts

```diff
@@ -42,9 +45,14 @@ describe('db init', () => {
         };
     }
 
+    function getDbModuleWithMockedLogger(dbConfig: DbConfig, mockedLogger: ReturnType<typeof mockLogger>) {
+        // @ts-ignore Passing mocked logger intentionally
+        return getDbModule(dbConfig, mockedLogger, [() => model]);
+    }
```

Comment:
Introduced such function to avoid adding `@ts-ignore` to each call to `getDbModule`. If you see better solution, I'm open to discuss.